src/guide/frontend/android.md

```diff
@@ -0,0 +1,340 @@
+   val apiUrl: String = "https://v4.passwordless.dev"
+)
+```
+
+### .well-known/assetlinks.json
+
+In your application's `AndroidManifest.xml`, you will then need to add the tag below under `manifest::application`:
+
+```xml
+<meta-data
+  android:name="asset_statements"
+  android:resource="@xml/assetlinks" />
+```
+
+In your application's `res/xml/assetlinks.xml`, you will then need to add the following content. This will tell our Android application where our `.well-known/assetlinks.json` file is hosted.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="assetlinks">https://yourexample.com/.well-known/assetlinks.json</string>
+</resources>
+```
+
+### Facet ID
+
+The `Facet ID` will be used at a later point in this guide to use as the `origin`.
+
+To obtain the Facet ID continue the steps below, the facet id typically looks like:
+
+`android:apk-key-hash:POIplOLeHuvl-XAQckH0DwY4Yb1ydnnKcmhn-jibZbk`
+
+1. Execute the following command in your terminal:
+
+::: tabs
+
+@tab Bash
+
+```bash
+# Linux, Mac OS, Git Bash, ...
+keytool -list -v -keystore ~/.android/debug.keystore | grep "SHA256: " | cut -d " " -f 3 | xxd -r -p | basenc --base64url | sed 's/=//g'
+```
+
+@tab Powershell
+
+```powershell
+# Run keytool command and extract SHA256 hash
+$keytoolOutput = keytool -list -v -keystore $HOME\.android\debug.keystore
+$sha256Hash = ($keytoolOutput | Select-String "SHA256: ").ToString().Split(" ")[2]
+
+# Remove any non-hex characters from the hash
+$hexHash = $sha256Hash -replace "[^0-9A-Fa-f]"
+
+# Convert the hexadecimal string to a byte array
+$byteArray = [byte[]]@()
+for ($i = 0; $i -lt $hexHash.Length; $i += 2) {
+  $byteArray += [byte]([Convert]::ToUInt32($hexHash.Substring($i, 2), 16))
+}
+
+# Convert the byte array to a base64 string
+$base64String = [Convert]::ToBase64String($byteArray)
+
+# Convert the base64 string to a base64url string
+$base64urlString = $base64String -replace '\+', '-' -replace '/', '_' -replace '=+$', ''
+
+Write-Output $base64urlString
+```
+
+:::
+
+2. The default password for the debug keystore is `android`. For your production keystore, enter your chosen password.
+
+3. Now append the result to `android:apk-key-hash:` to get the Facet ID:
+   `android:apk-key-hash:POIplOLeHuvl-XAQckH0DwY4Yb1ydnnKcmhn-jibZbk`
+
+## Configuration (Your back-end)
+
+### Obtaining the SHA-256 Certificate Fingerprints
+
+To configure your backend, you'll need to host a `.well-known/assetlinks.json` file at the root of your domain. This file contains a list of SHA-256 certificate fingerprints that are allowed to authenticate with your backend.
+
+This command will print detailed information about the keystore entry with the specified alias, including information about the certificate, its validity, and other details. It's commonly used in Android development to verify the properties of the debug keystore and the associated key used for signing applications during development.
```

Comment:
Would suggest changing "This command" to "The following command" to draw attention to the fact that you're describing what's in the tabs below.